### PR TITLE
Improvements: Removed static woo edd and elementor css and move to php file.

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- Removed static woo edd and elementor css and move to php file.
- Also deleted elementor.sass file.

### Types of changes
- Improvements

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards

### CODE REVIEW

**Review by premanshup — APPROVED** [review]

File: inc/class-astra-dynamic-css.php [hunk 1]
```diff
@@ -2945,6 +2945,26 @@ public static function load_cart_static_css() {
...
+				text-align: right;
+			}
+			.ast-header-break-point .ast-edd-site-header-cart-wrap .ast-edd-cart-menu-wrap {
+				width: 2em;
+				height: 2em;
+				font-size: 1.4em;
+				line-height: 2;
+				vertical-align: middle;
+				text-align: right;
+			}
```
Comment (imnavanath) [inline inc/class-astra-dynamic-css.php:2967]: Same CSS, so let's merge this selector - https://a.cl.ly/qGul61zk

File: inc/class-astra-dynamic-css.php [hunk 2]
```diff
@@ -2945,6 +2945,26 @@ public static function load_cart_static_css() {
...
+			  
+			.ast-header-break-point.ast-woocommerce-cart-menu.astra-hfb-header .ast-cart-menu-wrap, .ast-header-break-point.astra-hfb-header .ast-cart-menu-wrap {
+				width: 2em;
+				height: 2em;
+				font-size: 1.4em;
+				line-height: 2;
+				vertical-align: middle;
+				text-align: right;
+			}
+			.ast-header-break-point .ast-edd-site-header-cart-wrap .ast-edd-cart-menu-wrap {
```
Comment (imnavanath) [inline inc/class-astra-dynamic-css.php:2960]: @sagarb90 we have not used `-hfb` class selector here, so can you please verify if this CSS needed for both Old header footer & new HFB? If this is only for new HFB will it affect old header having EDD cart in menu?

File: inc/compatibility/class-astra-elementor.php [hunk 3]
```diff
@@ -87,6 +87,34 @@ public function enqueue_elementor_compatibility_styles( $dynamic_css, $dynamic_c
...
+						'width'  => 'auto',
+						'margin' => '0',
+						'float'  => 'none',
+					),
+					'.woocommerce div.product .elementor-element.elementor-products-grid .related.products ul.products li.product' => array(
+						'width'  => 'auto',
+						'margin' => '0',
+						'float'  => 'none',
+					),
+					'.elementor-image-gallery .gallery-icon' => array(
```
Comment (imnavanath) [inline inc/compatibility/class-astra-elementor.php:111]: Remove this CSS, as this already above on line 90 https://a.cl.ly/Qwubw6G2

File: inc/compatibility/class-astra-elementor.php [hunk 4]
```diff
@@ -87,6 +87,34 @@ public function enqueue_elementor_compatibility_styles( $dynamic_css, $dynamic_c
...
+					),
+					'.elementor-element .elementor-wc-products .woocommerce[class*=\'columns-\'] ul.products li.product' => array(
+						'width'  => 'auto',
+						'margin' => '0',
+						'float'  => 'none',
+					),
+					'.woocommerce div.product .elementor-element.elementor-products-grid .related.products ul.products li.product' => array(
+						'width'  => 'auto',
+						'margin' => '0',
+						'float'  => 'none',
```
Comment (imnavanath) [inline inc/compatibility/class-astra-elementor.php:109]: Same CSS, so let's merge these selectors - https://a.cl.ly/NQuKpEv0

File: inc/class-astra-dynamic-css.php (hunk 1 above)
Comment (sagarb90) [reply inc/class-astra-dynamic-css.php]: @imnavanath As this CSS is used only for new HFB so I have revert this CSS and keep as it is.

File: inc/class-astra-dynamic-css.php (hunk 2 above)
Comment (sagarb90) [reply inc/class-astra-dynamic-css.php]: @imnavanath As this CSS is used only for new HFB so I have revert this CSS and keep as it is.

File: inc/compatibility/class-astra-elementor.php (hunk 3 above)
Comment (sagarb90) [reply inc/compatibility/class-astra-elementor.php]: @imnavanath Done!

File: inc/compatibility/class-astra-elementor.php (hunk 4 above)
Comment (sagarb90) [reply inc/compatibility/class-astra-elementor.php]: @imnavanath Done!

**Review by imnavanath — APPROVED** [review]

**Pull request merged**